components/BottomNavigation/README.md

```diff
@@ -181,3 +181,82 @@ id<MDCTypographyScheming> typographyScheme = [[MDCTypographyScheme alloc] init];
 ```
 <!--</div>-->
 
+## Accessibility {#a11y}
+
+To help ensure your bottom navigation item is accessible to as many users as possible, please
+be sure to review the following recommendations:
+
+### `-accessibilityLabel`
+The label will be the title of the UITabBarItem. Currently you can't set this to a custom value.
+
+### Set `-accessibilityValue` 
+
+Set an appropriate
+[`accessibilityValue`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619583-accessibilityvalue)
+value if your item has a badge value. For example, an item with an inbox icon with a badge value for how many
+emails are unread. You should explicitly set the `accessibilityValue` when the badge value doesn't provide enough context. For example, in an inbox example simply having
+
+#### Objective-C
+```objc
+accessibilityValue = @"10";
+``` 
+#### Swift
+```swift
+accessiblityValue = "10"
+```
+doesn't provide enough context, instead the accessibility value should explain what the badge value symbolizes.
+
+#### Objective-C
+```objc
+accessibilityValue = @"10 unread emails";
+```
+
+
+#### Swift
+```swift
+accessibilityValue = "10 unread emails"
+```
+
+The default value if there is a badge value and you haven't set any `accessibilityValue` will be that the `accessibilityValue` is the `badgeValue`.
+
+#### Objective-C
+```objc
+tabBarItem.accessibilityValue = @"10 unread emails";
+```
+#### Swift
+```swift
+tabBarItem.accessibilityValue = "10 unread emails"
+```
+
+### Minimum touch size
+
+Make sure that your bottom navigation bar respects the minimum touch area. The Google Material spec calls for [touch areas that should be least 48 points high and 64 wide](https://material.io/design/layout/spacing-methods.html#touch-click-targets). 
+
+##### Objective-C
+```objc
+- (void)viewWillLayoutSubviews {
+  [super viewWillLayoutSubviews];
+  CGSize size = [_bottomNavigationBar sizeThatFits:self.view.bounds.size];
+  CGRect bottomNavBarFrame = CGRectMake(0, 
+                                        CGRectGetHeight(self.view.bounds) - size.height,
+                                        size.width,
+                                        size.height);
+  _bottomNavigationBar.frame = bottomNavBarFrame;
+}
+
+```
+
+##### Swift
+```swift
+override func viewWillLayoutSubviews() {
+  super.viewWillLayoutSubviews()
+  let size = bottomNavBar.sizeThatFits(view.bounds.size)
+  let bottomNavBarFrame = CGRect(x: 0,
+                                 y: view.bounds.height - size.height,
+                                 width: size.width,
+                                 height: size.height)
+  bottomNavBar.frame = bottomNavBarFrame
+}
+
+```
+As long as you use `sizeThatFits` you'll be fine. If you explicitly set the height the component will respond to that new height and may not respect the recommended touch area.
```

Comment:
We're mentioning `sizeThatFits`, but what if the provided size is smaller than the `sizeToFit`?

Reply:
So if they explicitly put` _bottomNavBar.frame.size.height = 20;` then the frame will be 20 and it won't respect touch areas.

Reply:
Sorry, I wasn't clear. I mean what if they call `sizeThatFits:CGSizeMake(64, 80)`? Will it respect that size or how does it work? If our component won't work with autolayout, please file an Issue so we can track the work. It might make sense to add it to the README as well, since it's a pretty big miss on our part to not support autolayout.

Reply:
<img width="410" alt="screen shot 2018-07-11 at 3 44 19 pm" src="https://user-images.githubusercontent.com/7131294/42595785-522c256c-8521-11e8-86cd-a12f3ffb609d.png">

Reply:
That's what you get if you pass 64,80 to size that fits.

Reply:
Ok, so using `sizeThatFits` shouldn't be recommended, right?

Reply:
I don't believe we support autolayout.

Reply:
Seems we don't. #2940 😞

Reply:
So, should we change the guidance here? It seemed like using `-sizeThatFits:` doesn't guarantee sufficient height (or width) for touch targets.  Should we recommend they provide some minimum height at least? 

Maybe it's better to just drop this particular recommendation and just leave the bit about making sure touch targets are big enough?